torch/_inductor/config.py

```diff
@@ -818,15 +846,12 @@ class cpp:
 
     simdlen: Optional[int] = None
     min_chunk_size = int(os.environ.get("TORCHINDUCTOR_CPP_MIN_CHUNK_SIZE", "4096"))
-    cxx = (
+
+    cxx: Tuple[None, Literal["g++-12", "g++-11", "g++-10", "clang++", "g++.par"]] = (
         None,  # download gcc12 from conda-forge if conda is installed
```

Comment:
I just copied it from the comment , but I think probably removing them is best and just making it a string because it kinda looks like a list of compilers that we support if we make it comprehensive.